Install into "amaya" directory instead of "Amaya-$APP_VER", to make packaging
easier.

Index: amaya-9.52.0.debian.1/configure.in
===================================================================
--- amaya-9.52.0.debian.1.orig/configure.in	2006-10-18 21:20:38.381389000 +0100
+++ amaya-9.52.0.debian.1/configure.in	2006-10-18 21:20:47.721972750 +0100
@@ -1426,7 +1426,7 @@
 	VPATHOPT="VPATH"
 fi
 
-AMAYA_INSTALLDIR="Amaya-${APP_VER}"
+AMAYA_INSTALLDIR="amaya"
 
 dnl
 dnl Exports the current values of the variables.
